csharp_dotnetcore/3.Welcome-Users/readme.md

```diff
@@ -0,0 +1,45 @@
+﻿This sample shows how to create a simple echo bot that welcomes user when they join the conversation. The welcoming pattern shown in this bot is applicable for peronsal (1:1) conversation with bots.
+
+## To try this sample
+- Clone the repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+```
+ ### Visual Studio
+- Navigate to the samples folder (`BotBuilder-Samples\csharp_dotnetcore\3.Welcome-Users`) and open 3.Welcome-Users.csproj in Visual Studio 
+- Hit F5
+ ### Visual Studio code
+- Open `BotBuilder-Samples\csharp_dotnetcore\3.Welcome-Users` folder
+- Bring up a terminal, navigate to BotBuilder-Samples\csharp_dotnetcore\3.Welcome-Users
+- Type 'dotnet run'.
+
+# Testing the bot using Bot Framework Emulator
+Microsoft Bot Framework Emulator is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+- Install the Bot Framework Emulator from here
+
+## Connect to bot using Bot Framework Emulator V4
+Launch Bot Framework Emulator
+File -> Open Bot Configuration and navigate to samples\3.Welcome-Users folder
+Select Welcome-Users.bot file
+
+# ConversationUpdate Activity Type
+The [ConversationUpdate](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-activity-spec?view=azure-bot-service-3.0#conversation-update-activity) activity type describe a change in a conversation's members, for example when a new user (and/or) a bot joins the conversation. It is up to the channel to send this activity when a user joins the conversation. Therefore, the why this actvity is sent changes from channel to channel. It is recomended that you test your bot behaivor on the target channel. 
+
+Bots that are added directly by a user, are mostly personal (1:1) conversation bots. It is a best practice to send a welcome message to the General channel to introduce the bot tell a bit about its functionality. To do this, ensure that your bot responds to the `ConversationUpdate` message. Use the `membersAdded` field to identify the list of channel participants (bots or users) that were added to the conversation.
```

Comment:
grammer. Please cleanup this block. remove the weird ",". 
What's the "General" channel? 
Why are we using back ticks here (`) when everywhere else it's quotes or regular ticks?